AUTHORS

```diff
@@ -56,27 +60,34 @@ David Donchez                           donch@dailymotion.com
 Deepesh Pathak                          deepshpathak@gmail.com
 Devarshi Sathiya                        devarshisathiya5@gmail.com
 Dharma Bellamkonda                      dharma.bellamkonda@gmail.com
+Didier Durand                           durand.didier@gmail.com
 Diego Casati                            diego.casati@gmail.com
 Dmitry Kharitonov                       geakstr@me.com
 Dom Del Nano                            ddelnano@gmail.com
+elfadel                                 bonfohelfadel@gmail.com
```

Comment:
Maybe add a `.mailmap` entry with the real name as per 82d03524a83262125177641f87f2d9bc24986aca's `Signed-off-by`:

```
El-Fadel Bonfoh <elfadel@accuknox.com>
```

Reply:
Sorry, I think my suggested `.mailmap` entry was wrong. It should have said:

```
El-Fadel Bonfoh <elfadel@accuknox.com> <bonfohelfadel@gmail.com>
```